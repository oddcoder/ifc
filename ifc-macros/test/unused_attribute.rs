use ifc_macros::ifc_block;

fn main() {
    ifc_block!{
        #[IFC(Unsafe)]
        let x : usize = 5;
        // This is little hack that triggers compiler errors
        // This way we can detect compiler warnings that
        // should trigger
    };
    compile_error!("Warning_Error");
}