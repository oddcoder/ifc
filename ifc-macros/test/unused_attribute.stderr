warning: Unused attribute
 --> $DIR/unused_attribute.rs:5:15
  |
5 |         #[IFC(Unsafe)]
  |               ^^^^^^
  |
  = help: Try removing it

error: Warning_Error
  --> $DIR/unused_attribute.rs:11:5
   |
11 |     compile_error!("Warning_Error");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
